go/chat/bots/commands.go

```diff
@@ -249,29 +260,8 @@ func (b *CachingBotCommandManager) ListCommands(ctx context.Context, convID chat
 		return res, alias, nil
 	}
 
-	var teamBotSettings map[keybase1.UID]keybase1.TeamBotSettings
 	cmdDedup := make(map[string]bool)
 	for _, ad := range s.Advertisements {
-		// If the advertisement is by a restricted bot that will not be keyed
-		// for commands, filter the advertisement out.
-		if ad.UntrustedTeamRole.IsRestrictedBot() {
-			if teamBotSettings == nil {
-				teamBotSettings, err = b.G().InboxSource.TeamBotSettingsForConv(ctx, b.uid, convID)
-				if err != nil {
-					return res, alias, err
-				}
-			}
-			kuid := keybase1.UID(ad.UID.String())
-			if !teamBotSettings[kuid].Cmds {
```

Comment:
where is the equivalent of this in the new code?

Reply:
TestBotCommandManager shoudl be updated if we no longer process these restrictions in go

Reply:
> where is the equivalent of this in the new code?

oh i see here https://github.com/keybase/client/pull/22096/commits/5b4677d9861cedf61ae747c7f632df5150991ae5 now

Reply:
Counts for being red